tests/lava/proc/lif/test_models.py

```diff
@@ -386,6 +386,31 @@ def test_bitacc_pm_impulse_dv(self):
         self.assertListEqual(expected_v_timeseries, lif_v)
         self.assertListEqual(expected_float_v, lif_v_float.tolist())
 
+    def test_bitacc_pm_eff_bias(self):
+        """
+        Tests fixed point LIF ProcessModel's scaling of threshold.
+        """
+        shape = (1,)  # a single neuron
+        num_steps = 1
+        # Set up high threshold and high bias current to check for potential
+        # overflow in effective bias
+        lif = LIF(shape=shape,
+                  du=0,
+                  dv=0,
+                  bias_mant=2 ** 12 - 1,
+                  bias_exp=5,
+                  vth=2 ** 17)
+
+        rcnd = RunSteps(num_steps=1)
+        rcfg = LifRunConfig(select_tag='fixed_pt')
+
+        lif.run(condition=rcnd, run_cfg=rcfg)
+        lif_v = lif.v.get()[0]
+        lif.stop()
+
+        # Check if lif_v is positive
+        self.assertGreater(lif_v, 0)
```

Comment:
For explicit bias, decay, and threshold values, you should be able to test against the exact voltage value that you expect here.

Reply:
Addressed in 8fa7f87.